homeassistant/components/victron_ble/strings.json

```
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "flow_title": "{title}",
+    "step": {
+      "user": {
+        "data": {
+          "address": "The Bluetooth address of the Victron device."
+        },
+        "data_description": {
+          "address": "This Bluetooth address is automatically discovered. You may view a device's Bluetooth address in the VictronConnect app under Settings > Product Info > Instant readout details > Encryption data > MAC Address."
+        }
+      },
+      "access_token": {
+        "title": "{title}",
+        "data": {
+          "access_token": "The instant readout encryption key of the Victron device."
+        },
+        "data_description": {
+          "access_token": "The instant readout encryption key may be found in the VictronConnect app under Settings > Product Info > Instant readout details > Encryption data > Encryption Key."
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "invalid_access_token": "Invalid Instant Readout Encryption Key",
```

We should sentence-case the whole string as these words need to be translated:

```suggestion
      "invalid_access_token": "Invalid instant readout encryption key",
```
This makes it consistent with the spelling in the `access_token` strings.